packages/core/plop/stories/index.js

```diff
@@ -1,48 +1,46 @@
-const fs = require("fs");
+import fs from "fs";
 
-module.exports = plop => {
-  plop.setGenerator("Stories", {
-    description: "New stories files for existing component",
-    prompts: [
-      {
-        type: "input",
-        name: "componentName",
-        message:
-          "What is the name of your component? (please use pascal case - for example: MultiWords for the component multi word)",
-        validate(input) {
-          return fs.existsSync(`./src/components/${input}`)
-            ? true
-            : "A component with this name does not exist in our library. Please check if there is a typing error in the name of the component you wrote.";
-        }
+export default {
+  description: "New stories files for existing component",
+  prompts: [
+    {
+      type: "input",
+      name: "componentName",
+      message:
+        "What is the name of your component? (please use pascal case - for example: MultiWords for the component multi word)",
+      validate(input) {
+        return fs.existsSync(`./src/components/${input}`)
+          ? true
+          : "A component with this name does not exist in our library. Please check if there is a typing error in the name of the component you wrote.";
       }
-    ],
-    actions: [
-      {
-        type: "add",
-        path: "src/components/{{properCase componentName}}/__stories__/{{properCase componentName}}.mdx",
-        templateFile: "plop/general/component-stories-mdx.txt"
-      },
-      {
-        type: "modify",
-        path: "src/components/{{properCase componentName}}/__stories__/{{properCase componentName}}.mdx",
-        pattern: /@componentName@/g,
-        template: "{{properCase componentName}}"
-      },
-      {
-        type: "add",
-        path: "src/components/{{properCase componentName}}/__stories__/{{properCase componentName}}.stories.tsx",
-        templateFile: "plop/general/component-stories-js.txt"
-      },
-      {
-        type: "add",
-        path: "src/components/{{properCase componentName}}/__stories__/{{properCase componentName}}.stories.helpers.tsx",
-        templateFile: "plop/general/component-stories-helpers-js.txt"
-      },
-      {
-        type: "add",
-        path: "src/components/{{properCase componentName}}/__stories__/{{properCase componentName}}.stories.module.scss",
-        templateFile: "plop/general/component-stories-scss.txt"
-      }
-    ]
-  });
+    }
+  ],
+  actions: [
+    {
+      type: "add",
+      path: "src/components/{{properCase componentName}}/__stories__/{{properCase componentName}}.mdx",
+      templateFile: "plop/general/component-stories-mdx.txt"
+    },
+    {
+      type: "modify",
+      path: "src/components/{{properCase componentName}}/__stories__/{{properCase componentName}}.mdx",
+      pattern: /@componentName@/g,
+      template: "{{properCase componentName}}"
+    },
```

Comment:
same, why are we adding and then modifying? 🤔 
maybe out of scope for current PR, but we should check some day

Reply:
I think it will be redundant once we remove the other generators, I'll check back